contracts-periphery/package.json

```diff
@@ -2,11 +2,20 @@
   "name": "@umbra/contracts-periphery",
   "version": "0.0.1",
   "license": "MIT",
+  "files": [
```

Comment:
_(arbitrary comment location to start a thread)_

Make sure to resolve conflicts and get CI passing too 🙂

Reply:
CI should be passing now